docs/user/cli.md

```diff
@@ -329,8 +329,9 @@ Finish the installation rebooting the system by default
 Possible values:
 
 - `stop`:
-  Stop at the end of the installation
-- `reboot`:
+  Stop the installation at its end, at the "Congratulations! [Reboot]" screen,
+  leaving the backend running
```

Comment:
But now I am wondering... `agama finish stop` seems to be a no-op then? What am I missing?

Reply:
That is exactly the case, what about renaming it to **agama.finish=no**, probably someone would expect then also the **agama.finish=yes** but by default reboot will be applied in case not a valid option an called from unattended installation, in the CLI it will report an error

Reply:
I am not sure if we sould mention the UI at all, I mean what is shown in the UI is because the installer is in the installed phase but maybe @imobachgs or @dgdavid have an stronger opinion